src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzerTests.cs

```diff
@@ -63,22 +64,22 @@ Imports Microsoft.CodeAnalysis.Diagnostics
 
 <DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)>
 Class MyAnalyzer
-	Inherits DiagnosticAnalyzer
```

Comment:
I was sure I reverted this change before doing the commit... Let me know if you prefer not to have these included here, that might take me some time to revert each line.

Reply:
It is fine given we are getting rid of tabs.